pkg/helpers/aws.go

```diff
@@ -347,6 +347,8 @@ func CloudWatchLogsSubscribe(ctx context.Context, cw cloudwatchlogsiface.CloudWa
 func CloudWatchLogsStream(ctx context.Context, cw cloudwatchlogsiface.CloudWatchLogsAPI, w io.WriteCloser, group, stream string, opts structs.LogsOptions) error {
 	defer w.Close()
 
+	fmt.Printf("CloudWatchLogsStream group=%q stream=%q", group, stream)
```

Comment:
Should generate some sort of random ID at the top here and then thread it through all of the following logs